packages/runtime/lib/management-api.js

```diff
@@ -116,56 +116,19 @@ async function createManagementApi (configManager, runtimeApiClient, loggingPort
     })
 
     app.get('/logs/live', { websocket: true }, async (connection, req) => {
-      const handler = (message) => {
-        for (const log of message.logs) {
-          connection.socket.send(log)
-        }
-      }
-
-      loggingPort.on('message', handler)
-      connection.socket.on('close', () => {
-        loggingPort.off('message', handler)
-      })
-      connection.socket.on('error', () => {
-        loggingPort.off('message', handler)
-      })
-      connection.socket.on('end', () => {
-        loggingPort.off('message', handler)
-      })
+      const startLogIndex = req.query.start ? parseInt(req.query.start) : null
+      pipeLiveLogs(connection, req.log, startLogIndex)
     })
 
-    app.get('/logs/history', { websocket: true }, async (connection, req) => {
-      const runtimeTmpFiles = await readdir(runtimeTmpDir)
-      const runtimeLogFiles = runtimeTmpFiles
-        .filter((file) => file.startsWith('logs'))
-        .map((file) => join(runtimeTmpDir, file))
-        .sort()
-
-      if (runtimeLogFiles.length === 0) {
-        connection.end()
-        return
-      }
+    app.get('/logs/latest/index', async () => {
```

Comment:
why can't we just return all the indexes for this?

Reply:
we can. but for now it would be integers from 0 to the last index. Not sure why.